week-4/implement/card-validator.js

```diff
@@ -0,0 +1,36 @@
+let cardNumber = '1009000100000108';
+cardNumber = cardNumber.replace(/\D/g, '');  // to remove any non digits.
+function creditCardLength(cardNumber) { // the function is to check card number length.
+    let theLength = (Math.log(Math.abs(cardNumber)+1) * 0.43429448190325176 | 0) + 1;
```

Comment:
This seems like complicated logic to get the length of the credit card number. Can you think of a simpler way to get the length of the string?